### DISCUSSION TITLE
How to use GitHub Desktop

### DISCUSSION THREAD

**Discussion by zaalsabb** [chulminy/CIVE497-CIVE700#108] [General]
Hi everyone!

I recommend that you download GitHub desktop so that you can store the most up-to-date copy of the lectures and tasks on your computer. Here's how to use it:

1) Download the GitHub desktop for your system here, and then choose to install it:
https://desktop.github.com/

2) Once you finish installing, go to File -> Clone Repository
![image](https://user-images.githubusercontent.com/46609733/104767158-b668b580-5739-11eb-8d04-e02feace913f.png)

3) Choose the URL option, then enter the course's url (https://github.com/chulminy/CIVE497-CIVE700), and where you would like to install it on your system, and then click clone.
![image](https://user-images.githubusercontent.com/46609733/104767392-0ba4c700-573a-11eb-869b-f0193eae5b03.png)

4) Once it is finished cloning, the repository is now stored on your computer. You can use the fetch origin feature to get the most up-to-date version that may include new tasks or lectures.
&nbsp;
&nbsp;
**Be careful not to overwrite your own solution files when you do the tasks, so you may want to store your task folders in a different location!**
&nbsp;
&nbsp;
![image](https://user-images.githubusercontent.com/46609733/104768198-3d6a5d80-573b-11eb-97d0-d070bae4e978.png)

**Comment by chulminy** [comment]
Great information!
